django_recaptcha/static/django_recaptcha/js/widget_v2_checkbox.js

```diff
@@ -0,0 +1,36 @@
+(function () {
+
+  function main() {
+    for (const captchaElement of document.querySelectorAll(".g-recaptcha")) {
+
+      const widgetUuid = captchaElement.getAttribute("data-widget-uuid");
+      if (!widgetUuid) {
+        console.warn("reCAPTCHA widget with missing UUID");
+        break;
+      }
```

Comment:
I don’t see in what scenario this could happen. If it’s possible, would recommend adding server logs for this.